lib/main.js

```diff
@@ -18,43 +18,54 @@ function HTMLUglify(config) {
 
   var salt = config.salt || 'use the force harry';
```

Comment:
I could easily convert `Math.random()` to a string. 

Thoughts on which would be the fastest? Math.random() or the crypto lib?

Currently this htmlUglify gets instantiated on each email sent.